src/main/java/lottogame/domain/lotto/LottoGenerator.java

```diff
@@ -0,0 +1,41 @@
+package lottogame.domain.lotto;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class LottoGenerator {
+    private static final int LOTTO_MIN = 1;
+    private static final int LOTTO_MAX = 45;
+    private static List<Integer> numbers;
```

Comment:
```suggestion
    private static final List<Integer> NUMBERS;
```
변경이 없다면 `final`을 붙여도 좋겠군요!
https://djkeh.github.io/articles/Why-should-final-member-variables-be-conventionally-static-in-Java-kor/

Reply:
이것도 피드백 주셨던 LottoNumber를 만들게 되면서,
리팩토링 과정에서 Generator에서 제거하고 LottoNumber 안에서 final로 넣어 반영했습니다!! ☺